src/oc-id/.gitignore

```diff
@@ -22,6 +22,7 @@ vendor/ruby
 config/settings.local.yml
 config/settings/*.local.yml
 config/environments/*.local.yml# RailsConfig ignores
+config/newrelic.yml
```

Comment:
Should we do the same thing I [mentioned on account management](https://github.com/chef/account_management/pull/107#issuecomment-104389791)?